bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roochi2var.py

```diff
@@ -11,30 +11,28 @@
 ################################################################################
 
 
-r"""
-/**
-\class RooChi2Var
-\brief \parblock \endparblock
-\htmlonly
-<div class="pyrootbox">
-\endhtmlonly
-
-## PyROOT
-
-Constructor of RooChi2Var takes a RooCmdArg as argument also supports keyword arguments.
-
-\htmlonly
-</div>
-\endhtmlonly
-*/
-"""
-
-from ._utils import _kwargs_to_roocmdargs
+from ._utils import _kwargs_to_roocmdargs, cpp_signature
 
 
 class RooChi2Var(object):
+    """Constructor of RooChi2Var takes a RooCmdArg as argument also supports keyword arguments."""
+
+    @cpp_signature(
+        [
+            "RooChi2Var(const char *name, const char* title, RooAbsReal& func, RooDataHist& data,"
+            "        const RooCmdArg& arg1, const RooCmdArg& arg2=RooCmdArg::none(),const RooCmdArg& arg3=RooCmdArg::none(),"
+            "        const RooCmdArg& arg4=RooCmdArg::none(), const RooCmdArg& arg5=RooCmdArg::none(),const RooCmdArg& arg6=RooCmdArg::none(),"
+            "        const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none(),const RooCmdArg& arg9=RooCmdArg::none()) ;",
+            "RooChi2Var(const char *name, const char* title, RooAbsPdf& pdf, RooDataHist& data,"
+            "        const RooCmdArg& arg1, const RooCmdArg& arg2=RooCmdArg::none(),const RooCmdArg& arg3=RooCmdArg::none(),"
+            "        const RooCmdArg& arg4=RooCmdArg::none(), const RooCmdArg& arg5=RooCmdArg::none(),const RooCmdArg& arg6=RooCmdArg::none(),"
+            "        const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none(),const RooCmdArg& arg9=RooCmdArg::none()) ;",
+        ]
+    )
```

Comment:
```suggestion
    @cpp_signature(
        [
            "RooChi2Var(const char* name, const char* title, RooAbsReal& func, RooDataHist& data,"
            "        const RooCmdArg& arg1, const RooCmdArg& arg2=RooCmdArg::none(),const RooCmdArg& arg3=RooCmdArg::none(),"
            "        const RooCmdArg& arg4=RooCmdArg::none(), const RooCmdArg& arg5=RooCmdArg::none(),const RooCmdArg& arg6=RooCmdArg::none(),"
            "        const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none(),const RooCmdArg& arg9=RooCmdArg::none()) ;",
            "RooChi2Var(const char* name, const char* title, RooAbsPdf& pdf, RooDataHist& data,"
            "        const RooCmdArg& arg1, const RooCmdArg& arg2=RooCmdArg::none(),const RooCmdArg& arg3=RooCmdArg::none(),"
            "        const RooCmdArg& arg4=RooCmdArg::none(), const RooCmdArg& arg5=RooCmdArg::none(),const RooCmdArg& arg6=RooCmdArg::none(),"
            "        const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none(),const RooCmdArg& arg9=RooCmdArg::none()) ;",
        ]
    )
```

Doxygen doesn't like it if the `*` is not at the expected place.